spec/fixtures/files/youtube_playlist.xml

```diff
@@ -0,0 +1,631 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feed xmlns:yt="http://www.youtube.com/xml/schemas/2015" xmlns:media="http://search.yahoo.com/mrss/" xmlns="http://www.w3.org/2005/Atom">
+ <link rel="self" href="http://www.youtube.com/feeds/videos.xml?playlist_id=PLMEZyDHJojxMRL0ifs67xcfzTOiPrGMdF"/>
+ <id>yt:playlist:PLMEZyDHJojxMRL0ifs67xcfzTOiPrGMdF</id>
+ <yt:playlistId>PLMEZyDHJojxMRL0ifs67xcfzTOiPrGMdF</yt:playlistId>
+ <yt:channelId>UC5GOr7CavTbxAf18CKjukdg</yt:channelId>
+ <title>Ramones playlist - BEST songs</title>
+ <author>
+  <name>Radio Golcak</name>
+  <uri>https://www.youtube.com/channel/UC5GOr7CavTbxAf18CKjukdg</uri>
+ </author>
+ <published>2015-04-22T16:09:39+00:00</published>
+ <entry>
+  <id>yt:video:K56soYl0U1w</id>
+  <yt:videoId>K56soYl0U1w</yt:videoId>
+  <yt:channelId>UCbLByZawHMZx_TFppHXeNYw</yt:channelId>
+  <title>The Ramones - Blitzkrieg Bop (With Lyrics)</title>
+  <link rel="alternate" href="https://www.youtube.com/watch?v=K56soYl0U1w"/>
+  <author>
+   <name>EA0825</name>
+   <uri>https://www.youtube.com/channel/UCbLByZawHMZx_TFppHXeNYw</uri>
+  </author>
+  <published>2009-05-11T02:01:07+00:00</published>
+  <updated>2021-06-29T15:19:02+00:00</updated>
+  <media:group>
+   <media:title>The Ramones - Blitzkrieg Bop (With Lyrics)</media:title>
+   <media:content url="https://www.youtube.com/v/K56soYl0U1w?version=3" type="application/x-shockwave-flash" width="640" height="390"/>
+   <media:thumbnail url="https://i4.ytimg.com/vi/K56soYl0U1w/hqdefault.jpg" width="480" height="360"/>
+   <media:description>Edit:This video is out of synch i hope you understand what i was trying to do :P
+
+Hey ho, lets go hey ho, lets go!

```

Comment:
This is weird but are we allowed to distribute copyrighted song lyrics in our app, even like this? I think we'd be better off erring on the side of caution and stripping them out.

Reply:
Oh I saved the real feed straight from Youtube, but I can strip it just fine! It's not like the bad gem we use to parse RSS can even read those anyway :P

Reply:
I think I got em all @eviltrout